content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md

```diff
@@ -121,22 +121,22 @@ To troubleshoot, make sure the YAML files in your __data_ folder follow these ru
 
 {% data reusables.pages.yaml-linter %}
 
-For more information about Jekyll data files, see [Data Files](https://jekyllrb.com/docs/datafiles/) in the Jekyll documentation.
+For more information about Jekyll data files, see "[Data Files](https://jekyllrb.com/docs/datafiles/)" in the Jekyll documentation.
 
 ## Markdown errors
 
 This error means that your repository contains Markdown errors.
 
 To troubleshoot, make sure you are using a supported Markdown processor. For more information, see "[AUTOTITLE](/pages/setting-up-a-github-pages-site-with-jekyll/setting-a-markdown-processor-for-your-github-pages-site-using-jekyll)."
 
-Then, make sure the file in the error message uses valid Markdown syntax. For more information, see [Markdown: Syntax](https://daringfireball.net/projects/markdown/syntax) on Daring Fireball.
```

Comment:
@akordowski Similiarly to comment above, can we ensure the quotation marks remain absent? 💛